features/catalog/step_definitions/descriptor-creation.ts

```diff
@@ -19,34 +32,52 @@ When(
     });
     this.response = await apiClient.eservices.createDescriptor(
       this.eserviceId,
-      {
-        description: "Questo è un e-service di test",
-        audience: ["api/v1"],
-        voucherLifespan: 60,
-        dailyCallsPerConsumer: ESERVICE_DAILY_CALLS.perConsumer,
-        dailyCallsTotal: ESERVICE_DAILY_CALLS.total,
-        agreementApprovalPolicy: "AUTOMATIC",
-        attributes: {
-          certified: [],
-          declared: [],
-          verified: [],
-        },
-      },
       getAuthorizationHeader(this.token)
     );
   }
 );
 
-Given(
-  "l'utente ha già creato una versione in bozza per quell'e-service",
+Then(
+  "si ottiene status code 200 e il descrittore contiene i campi del precedente",
   async function () {
     assertContextSchema(this, {
-      token: z.string(),
-      eserviceId: z.string(),
+      descriptorId: z.string(),
+      response: z.object({
+        status: z.number(),
+        data: z.object({
+          id: z.string(),
+        }),
+      }),
     });
-    this.descriptorId = await dataPreparationService.createDraftDescriptor(
-      this.token,
-      this.eserviceId
+
+    const descriptor = (
+      await apiClient.producers.getProducerEServiceDescriptor(
+        this.eserviceId,
+        this.descriptorId,
+        getAuthorizationHeader(this.token)
+      )
+    ).data;
+
+    const newDescriptor = (
+      await apiClient.producers.getProducerEServiceDescriptor(
+        this.eserviceId,
+        this.descriptorId,
+        getAuthorizationHeader(this.token)
+      )
+    ).data;
+
+    assert.equal(this.response.status, 200);
+    assert.equal(descriptor.description, newDescriptor.description);
+    assert.equal(descriptor.voucherLifespan, newDescriptor.voucherLifespan);
+    assert.equal(
+      descriptor.dailyCallsPerConsumer,
+      newDescriptor.dailyCallsPerConsumer
+    );
+    assert.equal(descriptor.dailyCallsTotal, newDescriptor.dailyCallsTotal);
+    assert.equal(
+      descriptor.agreementApprovalPolicy,
+      newDescriptor.agreementApprovalPolicy
     );
+    assert.deepEqual(descriptor.attributes, newDescriptor.attributes);
```

Comment:
```suggestion
    assert.deepStrictEqual(descriptor, newDescriptor);
```
Surprisingly this works